modal/shared/images.py

```diff
@@ -0,0 +1,8 @@
+from modal import Image
+
+
+def add_datadog(image: Image):
+    return image.pip_install("datadog-api-client==2.21.0")
```

Comment:
mind expanding one why this helper would have api & not ddtrace in it?

Reply:
The ddtrace was accidental--replaced it with sentry instead